src/production/ProductionContext.ts

```diff
@@ -31,8 +31,7 @@ export function createProductionContext(ctx): IProductionContext {
   for (const module of modules) {
     if (module.isExecutable) {
       // reset the contents
-      module.contents = undefined;
-      module.read();
+      if (module.contents === undefined) module.read();
```

Comment:
does this still reset the contents? do we still need to reset the contents?
if not, comment should be removed